### PR TITLE
Release 7.9.3

### PR DESCRIPTION
**Pull request by tolusha**
Release 7.9.3

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,72 @@
+module github.com/eclipse/che-operator
```
Comment (davidfestal) [inline go.mod:1]: Any idea why this change has been introduced ?

Comment (tolusha) [reply go.mod:1]: Automatically generated is VS Code with go plugin and go 1.12 version
